helm-chart/binderhub/templates/configmap.yaml

```diff
@@ -19,6 +19,19 @@ data:
   binder.repo2docker-image: {{ .Values.repo2dockerImage | quote }}
   binder.hub-url: {{ .Values.hub.url | quote }}
   binder.base_url: {{ .Values.baseUrl | quote }}
+  {{ if .Value.use_as_federation_portal -}}
```

Comment:
values.yaml should use camelCase, while configmaps should use kebabcase ('-' separator)